### ISSUE TITLE
Cancel events

### ISSUE THREAD

**Issue by thaliawww-service** [thaliawww/concrexit#873]
In GitLab by @joren485 on May 20, 2019, 10:46

### One-sentence description

Add a way to cancel events.

### Motivation

When an event is canceled, it is indicated on the event page by the organiser. At the moment, organisers do this in multiple ways. For example, some delete the event, some put "[CANCELED]" in the title and some change the description. One consistent way to do this would be nice.

**Comment by thaliawww-service** [comment]
In GitLab by @se-bastiaan on May 20, 2019, 11:11

Is this really a problem for us to solve? It seems like this is more a people problem than a system problem. If we decide on a consistent way to do it we don't have to implement anything :shrug:

**Comment by thaliawww-service** [comment]
In GitLab by @se-bastiaan on May 20, 2019, 13:20

(at)rwechel

**Comment by thaliawww-service** [comment]
In GitLab by rwechel on May 22, 2019, 12:53

Past year we handled the policy of using the [CANCELED] prefix + push notification and email when an event was cancelled. So far this seemed to work fine. We've had like three events where this happened this year, we've encountered no problems or angry people.
I don't know of any events being deleted besides the CTF though. I would guess that the reason of deleting the CTF instead of using the prefix is because the event had no registrations yet.
While it's a pretty neat feature, my opinion is that it's not per say needed. Maybe a feature creep? :P
